src/chainparams.cpp

```diff
@@ -137,6 +137,16 @@ class CMainParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_CSV_LEGACY].nStartTime = 1462060800; // May 1st, 2016
         consensus.vDeployments[Consensus::DEPLOYMENT_CSV_LEGACY].nTimeout = 1525132800; // May 1st, 2018
 
+        // Deployment of Community Fund Accumulation
+        consensus.vDeployments[Consensus::DEPLOYMENT_COMMUNITYFUND_ACCUMULATION].bit = 7;
+        consensus.vDeployments[Consensus::DEPLOYMENT_COMMUNITYFUND_ACCUMULATION].nStartTime = 1525132800; // Apr 1st, 2018
+        consensus.vDeployments[Consensus::DEPLOYMENT_COMMUNITYFUND_ACCUMULATION].nTimeout = 1554076800; // Apr 1st, 2019
+
+        // Deployment of NTP Sync
+        consensus.vDeployments[Consensus::DEPLOYMENT_NTPSYNC].bit = 8;
+        consensus.vDeployments[Consensus::DEPLOYMENT_NTPSYNC].nStartTime = 1525132800; // Apr 1st, 2018
```

Comment:
1525132800 is may 1st 2018 not 1st Apr? Is this correct

Reply:
fixed 8f96226c59a02540c7e6dcbcecb35ad584514b80